config/settings.yml

```diff
@@ -1306,6 +1306,7 @@
         :ems_metrics_collector_worker_openstack_network: {}
         :ems_metrics_collector_worker_redhat: {}
         :ems_metrics_collector_worker_vmware: {}
+        :ems_metrics_openstack_default_service: "auto"
```

Comment:
So, `"auto"` simply means skip over the [`return` statement](https://github.com/ManageIQ/manageiq/pull/13918/files#diff-19a98746baed48741d11262fa64841a2R29) and default to `gnocchi`?

I think that just having `"gnocchi"` be the default here might be clearer.